### ISSUE TITLE
quick open folder with system default file manager

### ISSUE THREAD

**Issue by hzmming** [Ulauncher/Ulauncher#528]
I think when browser the file system, it is better to use 'ctrl + enter' to quick open folder with system default file manager, rather than use 'alt + enter' and select one more time.

(English is not my native language, hope you can understand what i say😊. )

**Comment by gornostal** [comment]
Thanks for the feature request.
I will consider.

**Comment by bertvandepoel** [comment]
I agree. This could be used together with something like xdg-open so it would just open in whatever the default file manager is. This is what several other launchers do and I was surprised Ulauncher doesn't yet. I would love to see this happen!
